pkg/kubectl/rollback.go

```diff
@@ -105,7 +104,7 @@ type DeploymentRollbacker struct {
 }
 
 func (r *DeploymentRollbacker) Rollback(obj runtime.Object, updatedAnnotations map[string]string, toRevision int64, dryRun bool) (string, error) {
-	d, ok := obj.(*extensions.Deployment)
+	d, ok := obj.(*extensionsv1beta1.Deployment)
```

Comment:
That's not sufficient. Extensions's deployments (and all controllers) are deprecated. We should switch all of the code to always use the one from `apps/v1`. This will require you to change the client code in that command as well.

Reply:
Similarly you'll need to update the implementations for StatefulSet, DaemonSet, etc.